assets/constants.ts

```diff
@@ -115,6 +115,22 @@ export default {
     "IncreasingUpdatedOn": "Most Recently Updated",
     "DecreasingUpdatedOn": "Least Recently Updated"
   },
+
+  // Sorting categories for contributor admin dashboard.
+  "CD_ADMIN_STATS_SORT_OPTIONS": {
+    "IncreasingLastActivity": "Most Recently Active",
+    "DecreasingLastActivity": "Least Recently Active",
+    "IncreasingPerformance": "Lowest to Highest Performance",
+    "DecreasingPerformance": "Highest to Lowest Performance",
+    "IncreasingAccuracy": "Lowest to Highest Accuracy",
+    "DecreasingAccuracy": "Highest to Lowest Accuracy",
+    "IncreasingSubmissions": "Lowest to Highest submitted Translations",
```

Comment:
Perhaps you should capitalize Submitted and Reviewed (in 4 places below) as well.